clang/docs/ReleaseNotes.rst

```diff
@@ -766,6 +766,7 @@ Bug Fixes to C++ Support
 - Fixed an assertion failure caused by mangled names with invalid identifiers. (#GH112205)
 - Fixed an incorrect lambda scope of generic lambdas that caused Clang to crash when computing potential lambda
   captures at the end of a full expression. (#GH115931)
+- Clang no longer rejects deleting a pointer of incomplete enumeration type. (#GH99278)
```

Comment:
Would prefer this mentions the CWG issue perhaps?  Or at least a touch of backstory?

Reply:
The CWG issue is NAD - the back story is just "regression"